google/ggrc-core
# GGRC-1785 Unify titles of fields for Tasks and Cycle Tasks

# Issue description

Unify titles of fields for Tasks and Cycle Tasks on New Task, Edit Tasks, Task Info pane, Task export/import, Global and Advanced Search

# Steps to test the changes

For tasks and cycle tasks, check fields' titles across the screens.
Screens:
Task info pane
Create task popup
Edit task popup
Task export/import
Task filter on Export page
Global search
Advanced search

# Solution description

Modified following files to unify the fields' titles across the mentioned screens.
src/ggrc-client/js/components/workflow/cycle-task-group-object-task/templates/cycle-task-group-object-task.stache
src/ggrc-client/js/models/business-models/cycle-task-group-object-task.js
src/ggrc-client/js/models/business-models/task-group-task.js
src/ggrc-client/js/templates/cycle_task_group_object_tasks/modal_content.stache
src/ggrc_workflows/models/cycle_task_group_object_task.py
src/ggrc_workflows/models/mixins.py
src/ggrc_workflows/models/task_group_task.py

# Sanity checklist




# PR Review checklist
